driver-reactive-streams/src/main/com/mongodb/reactivestreams/client/MongoClients.java

```diff
@@ -112,6 +113,11 @@ private static MongoClient create(final com.mongodb.internal.async.client.MongoC
         return new MongoClientImpl(asyncMongoClient);
     }
 
+    private static MongoDriverInformation wrapMongoDriverInformation(final MongoDriverInformation mongoDriverInformation) {
+        return (mongoDriverInformation == null ? MongoDriverInformation.builder() : MongoDriverInformation.builder(mongoDriverInformation))
+                .driverName("reactive-streams").build();
```

Comment:
I think this fixes a regression introduce when reactive streams was pulled into the driver repository.  

Also, simplifies the driver name as described in the ticket